veg2hab/vegkartering.py

```diff
@@ -460,8 +462,8 @@ def build_aggregate_habtype_field(row: gpd.GeoSeries) -> str:
             len(habitatkeuzes) == 1
         ), "Bij KeuzeStatus GEEN_OPGEGEVEN_VEGTYPE mag er maar 1 habitatkeuze zijn"
         assert (
-            vegtypeinfos == []
-        ), "Bij KeuzeStatus GEEN_OPGEGEVEN_VEGTYPE mag er geen VegTypeInfo zijn"
+            vegtypeinfos == [VegTypeInfo(percentage=100, SBB=[], VvN=[])],
```

Comment:
Dit ging er dus fout?

Reply:
Yes, hierdoor had je wel een keuze maar geen vegtypeinfo. Nu is het weer gelijkgetrokken